### DISCUSSION TITLE
Welcome to UrlHub Discussions!

### DISCUSSION THREAD

**Discussion by realodix** [realodix/urlhub#745] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ahmdakbrhbb** [comment]
hallo mas, saya mau tanya tanya dong, saya mau coba up repo mas yg url, tapi selalu gagal terus karena extension gmp di version 8.1 itu gada mas ada solusi kah?

**Reply by realodix** [reply]
Halo, tolong buat diskusi baru, kita bahas di sana. Terima kasih

**Comment by paulguzmanppm** [comment]
I got the project working with docker, can I submit the pr to contribute to that? What requirements do I need to meet to get it approved?
